src/ui/component/AccountList.tsx

```diff
@@ -22,11 +22,9 @@ const ADDRESS_TYPES = [
   },
 ];
 
-const AccountList = ({ renderTypeAction, renderAccountAction }) => {
-  const wallet = useWalletOld();
-  const [accounts, setAccounts] = useState<Record<string, DisplayedKeryring[]>>(
-    {}
-  );
+const AccountList = () => {
+  const wallet = useWallet();
+  const [accounts, setAccounts] = useState<DisplayedKeryring[]>([]);
```

Comment:
According to signature of `wallet.getAllClassAccounts` (see [here](https://github.com/RabbyHub/Rabby/blob/a3d9b83689b44c1781caac738de3ac1fa3b7f615/src/background/controller/wallet.ts#L1663)), typeof `accounts` should be `DisplayedKeryring[]` instead of `Record<string, DisplayedKeryring[]>`.

There's no type error emitted before changing `useWalletOld` to `useWallet` here, after we deprecate `useWalletOld`, this mismatch revealed, so we correct it here.